molecule/user_provided/prepare.yml

```diff
@@ -0,0 +1,81 @@
+---
+- name: Prepare
+  hosts: localhost
+  connection: local
+  gather_facts: false
+  become: yes
+  tasks:
+    - name: stopping service
+      ansible.builtin.service:
+        name: mariadb
+        state: stopped
+
+    - name: uninstalling
+      ansible.builtin.yum:
+        name:
+          - mariadb
+          - mariadb-server
+          - redis
+        state: absent
+
+    - name: deletings mysql
+      file:
+        path: /var/lib/mysql
+        state: absent
+
+    - name: Install Redis
+      ansible.builtin.yum:
+        name: redis
+        state: latest
+
+    - name: Enable and start Redis
+      ansible.builtin.service:
+        name: redis
+        enabled: yes
+        state: started
+
+    - name: Set Redis password
+      shell: |
+        echo "CONFIG SET requirepass "password"" | redis-cli
+
+    - name: Install mariadb
+      ansible.builtin.yum:
+        name:
+          - mariadb
+          - mariadb-server
+        state: latest
+
+    - name: Uncommenting bind-address
+      shell: |
+        sed -i 's/#bind_address=0.0.0.0/bind_address=0.0.0.0/g' /etc/my.cnf.d/mariadb-server.cnf
+
+
+    - name: Enable and start MariaDB
+      ansible.builtin.service:
+        name: mariadb
+        enabled: yes
+        state: started
+
+    - name: Create Trillian Database
+      shell: echo "CREATE DATABASE trillian;" | mysql
+
+    - name: Download schema
+      shell: |
+        curl -sSL  https://raw.githubusercontent.com/securesign/trillian/refs/heads/main/storage/mysql/schema/storage.sql > schema.sql 
```

Comment:
I would suggest always using `curl --retry` to ensure that network/infrastructure flakiness doesn't make this fail. However, since this is Ansible, you should use https://docs.ansible.com/ansible/latest/collections/ansible/builtin/get_url_module.html plus retry/delay https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_loops.html#retrying-a-task-until-a-condition-is-met